addons/website/static/src/js/widgets/dialog.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import Dialog from 'web.Dialog';
+
+Dialog.include({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _isBlocking(index, el) {
+        if (el.parentElement && el.parentElement.id === 'website_cookies_bar') {
```

Comment:
This is wrong, as I indicated in the task spec, the bug only affetcs the cookie bar when not in popup mode.

Reply:
Ok, now only applying when not `o_cookies_popup`.

It feels a bit strange though that on initial open the body scroll is blocked only if the popup is big enough to require a scrollbar itself - but becomes blocked after another popup is closed. (Also in 14.0)

Reply:
Hm, are you referring to the same bug as https://github.com/odoo/odoo/pull/83921#issuecomment-1031382129 ?
As QSM requested me to urgently review/r+ his PR this morning (to fix odoo.com), I first tried his branch and reading your comment, it actually seems like we are speaking about the same issue?
Funny timing. Anyway, if that's the case, QSM added it in his TODO list (and/or will create a task for it). We are still discussing about it. A test wouldn't be too much at some point to makes sure all those issues are fixed for good

Reply:
I intended to check if this still happens once #83921 is forward-ported to 15.0, but trying it on 14.0 I believe this is something different. Not sure it is an issue though: if the popup does not require a scrollbar, maybe it is ok to let the body scroll.

Reply:
IMHO, given the actual issues impacting the popup/scrollbar, this would either:
- Be a wanted behavior
- Not wanted, but totally fine
Not worth investigating or spending time on that indeed